docs/topics/migrations.txt

```diff
@@ -333,6 +336,12 @@ Note that this only works given two things:
   that your database doesn't match your models, you'll just get errors when
   migrations try to modify those tables.
 
+Note: if you have upgraded from Django 1.7 or before, and get
+'foreign key constraint' errors (for example, when you run manage.py test),
```

Comment:
Right, it might be mixing apps with and without migrations. It hit me as a problem when running test, hence the comment. Happy if you want to put it somewhere else. but I think it should be highlighted somewhere, else we'll get people like the two stackoverflows post I mentioned in the ticket "fixing" the problem by either downgrading to 1.7, or turning foreign key checking off in the database, neither of which sound like a good idea.